test/nimble_csv_test.exs

```diff
@@ -204,6 +204,87 @@ defmodule NimbleCSVTest do
     """
   end
 
+  NimbleCSV.define(CSVWithUnknownSeparator, separator: [",", ";", "\t"])
```

Comment:
Let's put all of this inside a `describe` block called "multiple seperators" or similar.